filebeat/module/mongodb/_meta/config.yml

```diff
@@ -0,0 +1,8 @@
+- module: mongodb
+  # All logs
```

Comment:
What does all logs mean here? Does mongodb only have 1 type of logs?

Reply:
Yes, that's all I found in their docs: https://docs.mongodb.com/manual/reference/log-messages/